.github/workflows/terraform.yml

```diff
@@ -26,15 +26,46 @@ jobs:
         uses: hashicorp/setup-terraform@v1
         with:
           cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
-
       - name: Terraform Format
         id: fmt
-        run: cd examples/create-env-cluster-topics/;terraform fmt
-
+        run: terraform fmt
+        continue-on-error: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.ENGINEERING_GITHUB_PERSONAL_ACCESS_TOKEN }}
       - name: Terraform Init
         id: init
-        run: cd examples/create-env-cluster-topics/;terraform init
-
+        run: terraform init
       - name: Terraform Validate
         id: validate
-        run: cd examples/create-env-cluster-topics/;terraform validate -no-color
+        run: terraform validate -no-color
+        if: github.event_name == 'pull_request'
+        with:
+          github-token: ${{ secrets.ENGINEERING_GITHUB_PERSONAL_ACCESS_TOKEN }}
+          script: |
+            const output = `#### Terraform Format and Style 🖌\`${{ steps.fmt.outcome }}\`
+            #### Terraform Initialization ⚙️\`${{ steps.init.outcome }}\`
+            #### Terraform Validation 🤖${{ steps.validate.outputs.stdout }}
+            #### Terraform Validation 🤖
+
+            \`\`\`
+            ${{ steps.validate.outputs.stdout }}
+            \`\`\`
+
+            *Pusher: @${{ github.actor }}, Action: \`${{ github.event_name }}\`, Working Directory: \`${{ env.tf_actions_working_dir }}\`, Workflow: \`${{ github.workflow }}\`*`;
+
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: output
+            })
+      - name: release
+        if: github.event_name == 'push' && github.base_ref == 'test'
```

Comment:
this is a mistake - `github.base_ref` is only available on `github.event_name == pull_request || pull_request_target`

https://docs.github.com/en/actions/learn-github-actions/contexts

![Contexts - GitHub Docs at 13 54 34 on 2023-01-24@2x](https://user-images.githubusercontent.com/1943284/214229894-46eced0c-0c36-4e1a-b922-f8df6040d2f2.png)

Reply:
please update/fix the source/upstream as well, thanks.

I think maybe the conditional isn't needed at all, I think the semantic-release plugin knows when to run/not run.